### DISCUSSION TITLE
Feature request - submit this software to winget package manager

### DISCUSSION THREAD

**Discussion by mjkl-gh** [Serial-Studio/Serial-Studio#117] [Ideas]
Microsoft released a package manager for windows called winget. I'm currently consolidating my developer config around installing most programs through winget. As I use your program almost daily, it would be really convenient to have it featured on winget as well. A lot of other (open source) projects are featured there as well

Documentation on how to be pulled into the package manager is at:
https://docs.microsoft.com/en-us/windows/package-manager/package/repository

I hope you will consider it!

**Comment by alex-spataru** [comment]
Hi! I will begin doing this as soon as I have some free time this weekend. I had a lot of things going on (thesis, new job, etc), but I look forward to continue improving Serial Studio, its availability and its documentation. Thanks a lot for the idea!
